ReCord_frontend.py

```diff
@@ -20,16 +31,102 @@ def my_form_post():
     """the view function which return the result page by using the input pass to the back end
```

Comment:
Everything below should be broken down into their own functions, intead of having a super long function here.